src/PokemonSearch.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { getPokemon } from './services/fetch-utils';
+
+
+export default function PokemonSearch() {
+  
+  const [pokemon, setPokemon] = useState([]);
+  const [pokemonInput, setPokemonInput] = useState('');
+  const [pokemonQuery, setPokemonQuery] = useState(pokemonInput);
+
+  async function grabAndStorePoke() {
+    
+    
+    const data = await getPokemon(pokemonQuery);
+    
+    setPokemon(data);
+  }
+  
+  useEffect(() => {
+    grabAndStorePoke();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  
+  
+  async function handleSubmit(e) {
+
+    e.preventDefault();
+    setPokemonQuery(pokemonInput);
+    
+    const data = await getPokemon(pokemonQuery);
+    setPokemon(data.results);
+    setPokemonQuery('');
+  
+  }
+  
+  
+  return (
+    <><div className="Pokemon" />
+      <form onSubmit={handleSubmit}>
+        <input onChange={e => setPokemonInput(e.target.value)} />
+        <button>Search Pokemon</button>
+      </form>
+      {
```

Comment:
adding a condition like `pokemon &&` here may prevent the app from crashing if there are no pokemon to map through